### DISCUSSION TITLE
can't install - gateway timeout

### DISCUSSION THREAD

**Discussion by nor001** [Genuitec/Copilot4Eclipse-community#51] [Help] [answered]
![image](https://github.com/user-attachments/assets/3f6d086d-3497-41dc-8e63-d4fee00b04df)

**Comment by wayneparrott** [comment, marked as answer]
1. Can you try installing 1 more time now?  I'm wondering if there was a network glitch or something like that.

2. Is there additional info in the Error Log that can share to help us determine what might be goofed?

Wayne
Copilot4Eclipse Team

**Comment by nor001** [comment]
I tried again and I managed to install it.

**Reply by wayneparrott** [reply]
Congrats! 
W

**Comment by shivanipacharne-eaton** [comment]
I am getting this below error

![image](https://github.com/user-attachments/assets/2097653a-0be1-4b2f-9d7d-8bff7129f5f9)

**Comment by wayneparrott** [comment]
Hi @shivanipacharne-eaton:
This issue is commonly caused when the JRE you are running Eclipse with is unable to verify a certificate authority of a certificate it receives from your organization's proxy server when setting up a secure connection to the genuitec download server.

1. Are you able to install other plugins from the Eclipse Marketplace? Try installing any plugin that is not provided by eclipse*.org. 

2. Check out this doc that provides more background and some approaches to resolving this network config issue: https://www.genuitec.com/products/copilot4eclipse/docs/resources/technotes/installation-pkix-fix

Let's know if this resolves your install issue.

Wayne
Copilot4Eclipse Team

**Reply by shivanipacharne-eaton** [reply]
Yes, I was going through this page. I have raised IT ticket for the same.
Also, I am unable to install other plugins from marketplace.

Thanks for your quick response.
